modules/renter/hostdb/scan.go

```diff
@@ -241,12 +229,11 @@ func (hdb *HostDB) threadedScan() {
 		if err != nil {
 			if build.DEBUG {
 				panic(err)
```

Comment:
not sure if we should do this in one go, or do it piecemeal, but slowly we can start replacing `if DEBUG panic()` with `build.Critical()`, because build.Critical will handle not-panicking correctly.